go.mod

```diff
@@ -23,6 +23,8 @@ require (
 	k8s.io/klog/v2 v2.80.1
 )
 
+replace github.com/openshift/api => github.com/stlaz/api v0.0.0-20230316150409-7dfb8e061a12
```

Comment:
Is this intended to stay within the PR?

Reply:
it's a fake bump, we can't merge until the API PR merges (which it did - I need to update the PR)